sdk/FEATURES.md

```diff
@@ -71,10 +71,19 @@ feature.
 
 ### ContainerOp
 ContainerOp defines the container spec for a pipeline component. It is implemented with Tekton's
-[steps](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#defining-steps) features under Tekton task. The generated 
+[steps](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#defining-steps) features under Tekton task. The generated
 Tekton task name is the same as the containerOp name whereas the step name is always called "main". The
 [sequential](/sdk/python/tests/compiler/testdata/sequential.py) python test is an example of how to use this feature.
 
+### Affinity, Node Selector, and Tolerations
+
+Affinity, Node Selector, and Tolerations are Kubernetes spec for selecting which node should run the component based on user-defined constraints. They are implemented with Tekton's [PipelineRunTaskSpec](https://github.com/tektoncd/pipeline/blob/master/docs/pipelineruns.md#specifying-task-run-specs) features under Tekton PipelineRun.
+The [affinity](/sdk/python/tests/compiler/testdata/affinity.py),
+[node_selector](/sdk/python/tests/compiler/testdata/node_selector.py), and
+[tolerations](/sdk/python/tests/compiler/testdata/tolerations.py) python tests are examples of how to use these features.
+
+This feature is recently implemented in Tekton and is available on the Tekton master branch and Tekton 0.13.0+.
```

Comment:
And is this feature also supported through our compiler now?

Reply:
Yes, @NikeNano updated this feature to use the new PipelineRunTaskSpec. Currently it works with the master branch of Tekton.